test/testutil.lua

```diff
@@ -405,10 +408,10 @@ end
 
 local architecture = uv.os_uname().machine
 
---- @param s 'x86_64'|'arm64'
+--- @param s 'x86_64'|'arm64'|'aarch64'
 --- @return boolean
 function M.is_arch(s)
-  if not (s == 'x86_64' or s == 'arm64') then
+  if not (s == 'x86_64' or s == 'arm64' or s == 'aarch64') then
```

Comment:
why is "aarch64" needed? not used in this PR.

and generally, we should have one token that means "arm", and then `is_arch` (or whatever calls it) should do the right thing for each platform.

Reply:
oooo sorry about that, I was testing the linux side using a raspberry pi and a desktop, I wanted to know if the limit was different between x86 and ARM.
linux reports `aarch64` not `arm64` (or at least reaspbian does), I can remove it since it turned out not to be needed.

Reply:
ok let's remove it?